CRM Solution Manager/CRM Solution/App.config

```diff
@@ -4,23 +4,6 @@
     <add key="OrgServiceUrl" value="https://test.api.crm8.dynamics.com/XRMServices/2011/Organization.svc" />
     <add key="DynamicsUserName" value="test@test.onmicrosoft.com" />
     <add key="DynamicsPassword" value="*****" />
-    <add key="GitUserName" value="v-raras" />
-    <add key="GitPassword" value="*****" />
```

Comment:
let git details be there in app.config not in configuration

Reply:
done.